pkgs/by-name/au/autograder/package.nix

```
@@ -0,0 +1,58 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  digital,
+}:
+
+python3.pkgs.buildPythonApplication {
```

- `python3.pkgs` has some issues, so it is recommanded to use `python3Packages` (I wont go into details, since they are quite complex, but you may read more here: #211340)

```suggestion
python3Packages.buildPythonApplication {
```

Applies also on other `python3.pkgs`!

---

Thanks, done!